cargo-shuttle/Cargo.toml

```diff
@@ -9,9 +9,9 @@ homepage = "https://www.shuttle.rs"
 rust-version = "1.70"
 
 [dependencies]
-shuttle-common = { workspace = true, features = ["models"] }
-shuttle-proto = { workspace = true }
-shuttle-service = { workspace = true, features = ["builder"] }
+shuttle-common = { workspace = true, features = ["models", "backend"] }
```

Comment:
Naming nit: should this feature still be named backend if it's used in the CLI?

Reply:
Hmm, what inside of `backend` does the cli need here?

Reply:
I was mistaken. A small refactor solved it, and reduced the dep tree by an extra 10 :D